lhotse/cut/base.py

```diff
@@ -506,6 +506,97 @@ def trim_to_supervisions(
             cuts.append(trimmed)
         return CutSet.from_cuts(cuts)
 
+    def trim_to_alignments(
+        self,
+        type: str,
+        max_pause: Seconds = 0.0,
```

Comment:
Can we make it `Optional[Seconds] = None` and if it's `None` set max_pause = -1 so that it splits all alignment items without merging, and when it's 0.0 it merges the consecutive ones? I think it's more transparent from the usage perspective.

Reply:
That makes sense. I'll change it.